### PR TITLE
Fix colours not being calculated when toggling multimirror. Cleanup some dead code

### PR DESCRIPTION
**Pull request by andybak**

### CODE REVIEW

File: Assets/Scripts/API/ApiMethods.MultiMirror.cs [hunk 1]
```diff
@@ -97,15 +97,6 @@ public static void WallpaperSymmetrySkewX(float x, float y)
             PointerManager.m_Instance.CalculateMirrors();
         }
 
-        [ApiEndpoint("symmetry.colorshift.enable",
```
Comment (mikeage) [inline Assets/Scripts/API/ApiMethods.MultiMirror.cs:100]: Is there any possibility someone could be using this? Do you need some kind of deprecation handling?

**Review by mikeage — COMMENTED** [review]
looks sensible, one Q

File: Assets/Scripts/API/ApiMethods.MultiMirror.cs (hunk 1 above)
Comment (andybak) [reply Assets/Scripts/API/ApiMethods.MultiMirror.cs:100]: Highly unlikely considering the number of known API users. And I don't want too commit to API stability for stuff that's only made it as far as beta.

Comment (mikeage) [reply Assets/Scripts/API/ApiMethods.MultiMirror.cs:100]: sure (it was more a technical question than a policy one :-) )

Comment (mikeage) [reply Assets/Scripts/API/ApiMethods.MultiMirror.cs:100]: merge away!

**Pull request merged**